src/GeneralStateTestsFiller/stEIP3607/nonEmptyAccount_contractDeployFiller.yml

```diff
@@ -0,0 +1,69 @@
+# Account with non-empty code attempts to send tx to create a contract
+
+nonEmptyAccount_contractDeploy:
+  _info:
+    comment: Account with non-empty code attempts to send tx to create a contract
+
+  env:
+    # A blockinfo section (obsolete). Do not change. (only coinbase and gasLimit could be changed)
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentGasLimit: 0xFF112233445566
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  # We prepare a state where the sending account already has code in it.
+  pre:
+     # Even if this account is not involved in transaction, it will be miner-reward touched upon generation of a state test
+    2adc25665018aa1fe0e6bc666dac8fc2697ff9ba:
+      balance: '0'
+      nonce: '1'
+      code: ''
+      storage: {}
+
+    # Account containing code from which we will try to send a transaction from
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '1000000000000000000'
+      nonce: '0'
+      code: ':raw 0x00'
+      storage: {}
+
+    # Init'd address keccak256(rlp(a94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0))
+    6295ee1b4f6dd65047762f924ecd367c17eabf8f:
+      balance: '0'
+      nonce: '0'
+      code: ''
+      storage: {}
+
+  # Transaction originating from account with code
+  transaction:
+    data:
+    - ':yul {
+         return (0, 32)
+       }'
+    gasLimit:
+    - '400000'
+    gasPrice: '10'
+    nonce: '0'
+    to: ''
```

Comment:
the 2 cases here is really when transaction has destination. 
and the second when it runs creation. (empty to) this are 2 different files